documents/knowledge/elements.yaml

```
@@ -4564,6 +4564,19 @@
       - "[[1,2,3]] : 1"
       - "[[1,1,1]] : 0"
 
+- element: "Þj"
+  name: Depth
+  description: Depth of ragged list
+  arity: 1
+  overloads:
+      lst: Depth
+  vectorise: false
+  tests:
+      - "[1, 2, 7] : 1"
+      - "3 : 0"
+      - "[1, [2, 3, [4]], [69]] : 3"
```

woops the list is the stack, so these need wrapped
```suggestion
      - "[[1, 2, 7]] : 1"
      - "[3] : 0"
      - "[[1, [2, 3, [4]], [69]]] : 3"
```